src/hyperfine/error.rs

```diff
@@ -62,6 +71,13 @@ impl<'a> fmt::Display for OptionsError<'a> {
             OptionsError::TooManyCommandNames(n) => {
                 write!(f, "Too many --command-name options: expected {} at most", n)
             }
+            OptionsError::DifferentCommandNameCountWithParameters(real, expected) => {
+                write!(
+                    f,
+                    "Current --command-name count is {}: expected {} as parameters",
```

Comment:
This will be shown to the user, so let's try to make it a bit more descriptive. Maybe:

"'--command-name' has been specified {} times. It has to appear exactly once, or exactly {} times (number of benchmarks)"